SQL/beestation_schema.sql

```diff
@@ -546,6 +546,21 @@ END
 $$
 DELIMITER ;
 
+-- Dumping structure for table ss13tgdb.SS13_criminal_records
+DROP TABLE IF EXISTS `SS13_criminal_records`;
+CREATE TABLE IF NOT EXISTS `SS13_criminal_records` (
+  `ckey` varchar(32) NOT NULL,
+  `crime` varchar(2048) NOT NULL,
+  `details` varchar(2048) NOT NULL,
+  `author` varchar(32) NOT NULL,
+  `time` datetime NOT NULL,
+  `fine` int(10) unsigned NOT NULL,
+  `author_ckey` varchar(32) NOT NULL,
+  `paid` int(10) unsigned NOT NULL,
+  `character_name` varchar(32) NOT NULL,
+  PRIMARY KEY (`ckey`)
```

Comment:
Using the ckey as the primary key is going to cause you a lot of problems. At least if you want the ability to have multiple records tied to the same key.